packages/atlas/src/components/_inputs/RadioInput/RadioInput.styles.ts

```diff
@@ -1,139 +1,163 @@
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 
-import { cVar, transitions } from '@/styles'
+import { cVar, square } from '@/styles'
 
-export type RadioButtonStyleProps = Partial<{
-  disabled: boolean
-  checked: boolean
-}> &
-  RadioButtonCaptionProps
-
-type RadioButtonCaptionProps = {
+type StyledRadioInputProps = {
   error?: boolean
+  disabled?: boolean
 }
 
-export const Input = styled.input`
-  margin: auto;
-  opacity: 0;
-  cursor: ${(props) => (props.disabled ? 'not-allowed' : 'pointer')};
-`
+export const StyledRadioInput = styled.div<StyledRadioInputProps>`
+  ${square('16px')};
 
-export type CustomRadioInputProps = {
-  disabled?: boolean
-  checked?: boolean
-  error?: boolean
-}
+  position: relative;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-clip: content-box;
+  padding: 4px;
+  box-sizing: border-box;
+  border: 1px solid ${({ error }) => cVar(error ? 'colorBorderError' : 'colorBorderAlpha')};
+  transition: all ${cVar('animationTransitionFast')};
+
+  ::before {
+    content: '';
+    display: block;
+    background-color: transparent;
+    top: -9px;
+    bottom: -9px;
+    left: -9px;
+    right: -9px;
+    border-radius: 50%;
+    position: absolute;
+    z-index: -1;
+    transition: all ${cVar('animationTransitionFast')};
+  }
 
-export const hoverState = (checked?: boolean) => {
-  return css`
-    border: ${checked ? `4px solid ${cVar('colorCoreBlue500')}` : `1px solid ${cVar('colorCoreNeutral200')}`};
+  ::after {
+    ${square('8px')};
 
-    &::before {
-      background-color: ${cVar('colorCoreNeutral700Lighten')};
-    }
-  `
-}
-export const activeState = (checked?: boolean) => {
-  return css`
-    border: ${checked ? `border: 4px solid ${cVar('colorCoreNeutral50')}` : `1px solid ${cVar('colorCoreNeutral50')}`};
-
-    &::after {
-      content: '';
-      position: absolute;
-      top: -4px;
-      bottom: -4px;
-      left: -4px;
-      right: -4px;
-      display: ${checked ? 'block' : 'none'};
-      border-radius: 50%;
-      border: 1px solid ${cVar('colorCoreNeutral50')};
+    content: '';
+    display: block;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    border-radius: 50%;
+  }
+`
+
+const hoverStyles = css`
+  + ${StyledRadioInput} {
+    ::before {
+      background-color: ${cVar('colorBackgroundStrongAlpha')};
     }
+  }
+`
+
+export const Input = styled.input<{ error?: boolean }>`
+  ${square('100%')};
 
-    &::before {
-      background-color: ${cVar('colorCoreNeutral800Lighten')};
+  position: absolute;
+  top: 0;
+  left: 0;
+  margin: auto;
+  opacity: 0;
+  cursor: ${(props) => (props.disabled ? 'not-allowed' : 'pointer')};
+  z-index: 999;
```

Comment:
We could use `zIndex.globalOverlay` here, but on the other hand, Is this value too big?